modules/solid_mechanics/src/materials/CombinedPlasticityStressUpdate.C

```diff
@@ -0,0 +1,258 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
```

Comment:
Can isotropic and kinematic be separated and then combined using the multiple inelastic materials input?  I would rather kinematic hardening was its own material.  If we do it like this, we duplicate all of the isotropic hardening classes.

Reply:
I agree that it would be good to avoid duplicating the isotropic hardening code, but I'm not sure what the best way to do that in practice is. We could potentially have it so that we optionally register the backstress for kinematic hardening models, but then we would be carrying around some extra baggage for the isotropic models.